modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/client/Iroha2Client.kt

```diff
@@ -111,9 +114,16 @@ open class Iroha2Client(
      * {@see Extractors}
      */
     suspend fun <T> sendQuery(queryAndExtractor: QueryAndExtractor<T>): T {
+        val page = sendQueryWithPagination(queryAndExtractor, null)
+        return page.data
+    }
+
+    suspend fun <T> sendQueryWithPagination(queryAndExtractor: QueryAndExtractor<T>, page: Pagination?): Page<T> {
```

Comment:
I'd suggest to leaving only one fun sendQuery, and change `page: Pagination?` to `page: Pagination? = null`

Reply:
If we leave only one function sendQuery then we will have to return page to every sendQuery call and thus this will require changes in all tests.
So sendQuery will return "page" not "page.data" since we may need to have additional information from page.

Reply:
okay